tests/Loader/ArrayLoaderTest.php

```diff
@@ -326,8 +326,8 @@ public function testLoadStepArguments(): void
                                         'type' => 'table',
                                         'rows' => [
                                             ['key', 'val'],
-                                            [1, 2],
-                                            [3, 4],
+                                            ['1', '2'],
+                                            ['3', '4'],
```

Comment:
We've said at some point that the table structure should only contain strings, which conflicts with an alias type defined in `ArrayLoader` (note that I removed the/an `|int`) and then of course this test (and therefore the change above).

I hope this change is correct.

Reply:
Indeed, we did agree that.